source/type/type.h

```diff
@@ -548,7 +548,7 @@ struct CommercialMode : public Header, Nameable{
 
 struct PhysicalMode : public Header, Nameable{
     const static Type_e type = Type_e::PhysicalMode;
-    double co2_emission = 0.;
+    double co2_emission = invalid_idx;
```

Comment:
2 values possibles for co2_emission:
XX : value defined
invalid_idx : if undefined co2_emission value 

Commented by l-vincent-l : please use std::numeric_limits::max()